### PR TITLE
[IMP] general: users doc rewrite

### PR DESCRIPTION
**Pull request by tiku-odoo**
Tasks: 
https://www.odoo.com/web#id=2895638&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=formhttps://www.odoo.com/web#id=3217323&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@brse-odoo 

When you have a moment can you review this doc on users? 

Thanks,
Tim

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:7]:
```suggestion
Odoo defines a *user* as someone who has access to a database. An
```
Optional for simplification

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:9]:
```suggestion
information each user can access, rules can be applied to each user. Users and access rights can be added and
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:23]:
```suggestion
To add new users, navigate to :menuselection:`Settings app --> Users --> Manage Users` and click on :guilabel:`Create`.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:29]:
```suggestion
Fill in the form with all required information. Under the :doc:`Access Rights
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:30]:
```suggestion
<users/access_rights>` tab, choose the group within each application the user can have access to.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:38]:
```suggestion
After filling out all the necessary fields on the page, click :guilabel:`Save`. An invitation
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:48]:
```suggestion
   If the company is on a monthly subscription plan, the database automatically updates to
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:49]:
```suggestion
   reflect the added users. If the company is on a yearly or multi-year plan, an expiration banner
```
I believe we were told not to use (s) and just use the -s. Feel free to double check me on that though

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:51]:
```suggestion
   the subscription. Alternatively, `send a support ticket <https://www.odoo.com/help>`_ to resolve the issue.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:63]:
```suggestion
   Public users are those visiting websites via the website's frontend.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:70]:
```suggestion
users have specific access rights pre-set (such as record rules and restricted menus) and usually do not
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:110]:
```suggestion
   If the company is on a monthly subscription plan, the database automatically updates to
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:111]:
```suggestion
   reflect the added users. If the company is on a yearly or multi-year plan, an expiration banner
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:113]:
```suggestion
   the subscription. Alternatively, `send a support ticket <https://www.odoo.com/help>`_ to resolve the issue.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:141]:
```suggestion
Sometimes, users might wish to reset their personal password for added security, so they
are the only ones with access to the password. Odoo offers two different reset options: one
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:142]:
```suggestion
initiated by the user to reset the password, and another where the administrator triggers a reset.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:150]:
```suggestion
completed by the individual user, and this setting is enabled by default.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:152]:
```suggestion
To change this setting, go to :menuselection:`Settings app --> Permissions`, activate
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:171]:
```suggestion
Go to :menuselection:`Settings app --> Users & Companies --> Users`, select the user out of the list and
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:194]:
```suggestion
password, and confirm the change by clicking on :guilabel:`Change Password`.
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:202]:
```suggestion
   account.
   
   If the odoo.com password needs to be changed, use the :ref:`send the password reset
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:203]:
```suggestion
   instructions <users/reset-password-email>`. Odoo.com passwords grant access to the *My
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:222]:
```suggestion
The :guilabel:`Allowed Companies` field can contain multiple companies. These will be the companies
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:223]:
```suggestion
the user has access to, and can make edits to, according to the set access rights. The
```

File: content/applications/general/users.rst
Comment (brse-odoo) [inline content/applications/general/users.rst:231]:
```suggestion
   If multi-company access is not configured correctly, it could lead to
   inconsistent multi-company behaviors. Because of this, only experienced Odoo users should make access
   rights changes to users for databases with a multi-company configuration. For technical explanations, refer to the
   developer documentation on :doc:`../../../developer/howtos/company`.
```

**Review by brse-odoo — APPROVED** [review]
@tiku-odoo I've finished my review of this PR, great job! I left some suggestions for you to accept/reject as you see fit -- mostly grammar and consistency-related stuff. Approving for the next round, let me know if you have any questions! Thanks.

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:82]:
```suggestion
select :guilabel:`Archive` from the resulting drop-down menu. Then, click :guilabel:`OK` from the :guilabel:`Confirmation` pop-up window that appears.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:85]:
```suggestion
   **Never** deactivate the main/administrator user (admin). Making changes to admin users can have a
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:90]: Error: too many users

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:94]:
```suggestion
the following message is displayed.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:106]:
```suggestion
  displayed in the message to validate the upsell quotation, and pay for the extra users.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:107]:
```suggestion
- :ref:`Deactivate users <users/deactivate>`, and reject the upsell quotation.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:113]:
```suggestion
   subscription. Alternatively, users can `send a support ticket <https://www.odoo.com/help>`_ to resolve the issue.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:123]:
```suggestion
Password management is an important part of granting users autonomous access to the database at all
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:128]:
```suggestion
   navigating to :menuselection:`Settings app --> Permissions section`, and entering the desired password length in
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:152]:
```suggestion
To change this setting, go to :menuselection:`Settings app --> Permissions section`, activate
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:159]:
```suggestion
On the login page, click :guilabel:`Reset Password` to initiate the password reset process, and have
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:171]:
```suggestion
Go to :menuselection:`Settings app --> Users & Companies --> Users`, select the user from the list,
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:172]:
```suggestion
and click on :guilabel:`Send Password Reset Instructions` on the user form. An email is
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:178]:
```suggestion
   appears.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:193]:
```suggestion
its form. Click on the :guilabel:`⚙️ Actions` icon, and select :guilabel:`Change Password` from, the resulting drop-down menu. Enter a
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:194]:
```suggestion
new password in the :guilabel:`New Password` column of the :guilabel:`Change Password` pop-up window that appears, and confirm the change by clicking :guilabel:`Change Password`.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:201]:
```suggestion
   This operation only modifies the password of the users locally, and does **not** affect their odoo.com
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:204]:
```suggestion
   If the odoo.com password needs to be changed, use the :ref:`send the password reset
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:206]:
```suggestion
   page, and other portal features.
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:208]:
```suggestion
After clicking :guilabel:`Change Password`, the page is redirected to an Odoo login page, where
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:216]:
```suggestion
The :guilabel:`Multi Companies` field on a user form allows an administrator to provide access to multiple companies
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:217]:
```suggestion
for existing users. To configure a multi-company environment for a user, navigate to the desired user by
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:218]:
```suggestion
going to: :menuselection:`Settings app --> Users section --> Manage users`. Then, select the user to open their user form, and
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:221]:
```suggestion
Under :guilabel:`Multi Companies` in the :guilabel:`Access Rights` tab, set the fields labeled :guilabel:`Allowed Companies` and
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:224]:
```suggestion
The :guilabel:`Allowed Companies` field can contain multiple companies. These are the companies
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:225]:
```suggestion
the user can access and edit, according to the set access rights. The
```

File: content/applications/general/users.rst
Comment (ksc-odoo) [inline content/applications/general/users.rst:226]:
```suggestion
:guilabel:`Default Company` is the company the user defaults to, upon logging in each time.
```

**Review by ksc-odoo — APPROVED** [review]
Great work on this, @tiku-odoo -- just finished my Final Review. Approving now. Once you've implemented the necessary changes, feel free to tag this for Tech Review. Thanks! 👍

**Comment by tiku-odoo** [comment]
@samueljlieber 

This doc is ready for your technical review. 

Thanks, 
Tim :+1:

File: content/applications/general/users.rst
Comment (samueljlieber) [inline content/applications/general/users.rst:104]: Make sure to remove the unused `users/password-email.png` image :)

File: content/applications/general/users.rst
Comment (samueljlieber) [inline content/applications/general/users.rst:199]: Double space
```suggestion
Go to :menuselection:`Settings app --> Users & Companies --> Users`, and select a user to access
```

File: content/applications/general/users.rst
Comment (samueljlieber) [inline content/applications/general/users.rst:249]: No need for bullet point in a single item
```suggestion
   :doc:`companies`
```

File: content/applications/general/users.rst
Comment (samueljlieber) [inline content/applications/general/users.rst:192]: Add a period
```suggestion
   :alt: Example of an email with a password reset link for an Odoo account.
```

File: content/applications/general/users.rst
Comment (samueljlieber) [inline content/applications/general/users.rst:246]: Add a period
```suggestion
   :alt: View of a user's form emphasizing the multi companies field in Odoo.
```

**Review by samueljlieber — APPROVED** [review]
Hi @tiku-odoo, nice job on the improvements to this doc! I am approving this PR with a couple of comments, please address them before merging :)
...
@robodoo delegate=tiku-odoo

**Comment by tiku-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
